lib/rooms/controller.go

```diff
@@ -118,6 +118,29 @@ func (c *RoomWSController) OnDisconnect(ctx *WSContext) {
 	}
 }
 
+func (c *RoomWSController) Leave(ctx *WSContext) {
+	isBroadcaster, err := c.roomRepo.IsBroadcaster(ctx.RoomId, ctx.SocketID)
+	if err != nil {
+		c.error(err)
```

Comment:
Shouldn't there be a 'return' statement in this if-statment (after c.error(err))?